config/i18n.json

```diff
@@ -503,6 +503,8 @@
     "SelectPower": "Minimum Power",
     "SelectPowerDescription": "Sets a minimum power level for the entire loadout (not individual pieces)",
     "SelectedPerkKey": "Selected perk",
+    "StatToggleIgnore": "Click this stat to ignore it.",
```

Comment:
Wasn't sure if should just use include/ignore or two translations as I have. Happy to change.